### PR TITLE
Tests/mutation nightly build fixes

### PR DESCRIPTION
**Pull request by sreekanthpuligadda**
…yker.

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: test/e2e/config/crossbrowser.conf.js [hunk 1]
```diff
@@ -46,28 +46,40 @@ const config = {
...
-      platform: 'macOS 10.13',
-      version: '11',
-      name: 'Safari-TEST',
-      tunnelIdentifier: 'reformtunnel',
-      extendedDebugging: true,
-      sharedTestFiles: false,
-      maxInstances: 1
-
-    },
+    //
```
Comment (udaydenduluri33) [inline test/e2e/config/crossbrowser.conf.js:59]: Delete this dead code please

File: test/integration/tests/utils/getCookie.ts [hunk 2]
```diff
@@ -1,29 +1,65 @@
...
-
-    await page.type('#username', username);
-    await page.type('#password', password);
-
-    await page.click('.button');
-    await page.waitForSelector('.hmcts-primary-navigation', { visible: true });
-  } catch (error) {
-    await browser.close();
-    throw error;
+  // try {
```
Comment (udaydenduluri33) [inline test/integration/tests/utils/getCookie.ts:26]: And also here, please remove dead code

**Review by udaydenduluri33 — APPROVED** [review]
LGTMv

**Pull request merged**